### DISCUSSION TITLE
Provide an issue Bounty system for better remuneration and motivation

### DISCUSSION THREAD

**Discussion by PhilipGarnero** [community/community#109165] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Most open source projects start hard and often stall after a few years.
It is often due to their contributors not having the time or the motivation to work on them anymore.
It is a shame because there are project widely used by the community with issues waiting to be fixed.

Why couldn't we have an issue bounty system ?
Let's say for a specific issue, any github user could add money that would be attributed to the project or the PR's author if the issue were to be fixed ?
If an issue has a few hundred votes, this could easily translate to a few hundreds bucks (or even more) for the contributors and incentivise them to close the aforementioned issue.
We could even imagine it attracting bounty hunter developers coming in to contribute to the issue for a share of the bounty.

Github could even take a small share helping the company to absorb the current opensource hosting fees.

I know for a fact that I would have already given a few hundreds buck across multiple issues if that was existing.

**Comment by l0uisgrange** [comment]
I totally agree, this is a very cool idea in every way!
